pages/glossary/distinguishing-styles.md

```diff
@@ -0,0 +1,11 @@
+--
+title: Distinguishing Styles
```

Comment:
For all of these things, specify that this needs to be the [computed value](https://www.w3.org/TR/css-cascade-3/#computed-value) of that particular CSS property.